.travis.yml

```diff
@@ -39,7 +39,7 @@ env:
     # END OF TRAVIS MATRIX
 
 before_install:
-  - sudo apt-get update ; sudo apt-get install -y curl apt-transport-https git
+  - sudo apt-get update ; sudo apt-get install -y curl apt-transport-https git netcat net-tools libpq-dev
```

Comment:
You're probably right, I can remove them, let me make sure there's no other reasonable use for them.